hazelcast/src/main/java/com/hazelcast/config/PersistentMemoryDirectoryConfig.java

```diff
@@ -0,0 +1,142 @@
+/*
+ * Copyright (c) 2008-2020, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.config;
+
+import java.util.Objects;
+
+/**
+ * Configuration class for directories that persistent memories are mounted at.
+ */
+public class PersistentMemoryDirectoryConfig {
+    private static final int NUMA_NODE_UNSET = -1;
+
+    private String directory;
+    private int numaNode = NUMA_NODE_UNSET;
+
+    /**
+     * Creates an instance with the {@code directory} specified.
+     *
+     * @param directory The directory where the persistent memory is
+     *                  mounted at
+     */
+    public PersistentMemoryDirectoryConfig(String directory) {
+        this.directory = directory;
```

Comment:
Can the directory be empty?

Reply:
Nope, if there is no intention to use persistent memory, `PersistentMemoryConfig` should have an empty directory config list. Updated accordingly.